templates/hooks/db_instance/sdk_update_pre_build_request.go.tpl

```diff
@@ -19,3 +19,15 @@
 		ackcondition.SetSynced(desired, corev1.ConditionFalse, &msg, nil)
 		return desired, requeueWaitUntilCanModify(latest)
 	}
+	if delta.DifferentAt("Spec.Tags") {
+		if err = rm.syncTags(ctx, desired, latest); err != nil {
+			return nil, err
+		}
+	}
+    if !delta.DifferentExcept("Spec.Tags") {
+        // Don't call ModifyDBInstance if none of the non-tag fields have
+        // changed, since calling ModifyDBInstance will cause a multi-minute
```

Comment:
I believe this issue is fixed here https://github.com/aws-controllers-k8s/rds-controller/blob/main/templates/hooks/db_instance/sdk_update_post_build_request.go.tpl#L15-L21, we can remove this comment looks like

Reply:
Yeah, lemme ditch this, nice catch :)

Reply:
done!